img/placeholder/icon-persona.svg

```diff
@@ -0,0 +1,4 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
+	 viewBox="0 0 80 80" style="enable-background:new 0 0 80 80;" xml:space="preserve" fill="#aaa">
+<circle class="st0" cx="40" cy="40" r="40"/>
+</svg>
```

Comment:
I think we need to have more discussions on this. 

1. `/img/` won't exist for consumers, so this really should be in `/libs/img`.
2. I'm starting to lean away from the idea of SVGs we use via img tags. Mostly because they're not themable. If we inject SVGs into the DOM, we can use fill and other tricks to style based on light / dark.
3. I don't think placeholder is the right term here. Seems like this should be either `/libs/img/icons` or `/libs/img/design-tokens`. I think icons makes more sense.

Reply:
good call. I've updated the path on these /libs/img etc. 
For our icon reference, I'm still using the src"" path for the image rather than injected SVG's. Let's discuss how we should tackle this part. We can probably automate some type of lib/img/sprite.html file w/ embedded/ID'd SVGs etc. We may already be doing something along these lines?